config/jobs/kubernetes/wg-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -107,17 +108,29 @@ periodics:
         org:kubernetes-client
         org:kubernetes-csi
         -repo:kubernetes-sigs/kind
+        -label:triage/accepted
```

Comment:
Bikeshed: @sftim posited excluding issues that have `triage/accepted` here: https://github.com/kubernetes/kubernetes/issues/103151#issuecomment-867564284

I'm a weak -1 on this specific change, because triaged issues fall stale too, and I'd rather describe the State of Today than bog us down on policy changes.

In the commit message, I did a bunch of label queries.  I can't tell one way or the other that this means anything, but here's some data.

    Kubernetes issues/PRs that have lifecycle/stale + triage/accepted:
    - total: 110 open, 51 closed
    - kubernetes/kubernetes: 79 open, 36 closed
    - kubernetes/website: 19 open, 10 closed
    - kubernetes-sigs/metrics-server: 6 open, 3 closed
    - kubernetes-sigs/kustomize: 4 open, 2 closed
    - kubernetes-sigs/kubebuilder: 3 open, 2 closed
    - kubernetes/kubectl: 4 open, 0 closed
    - kubernetes/ingress-nginx: 1 open, 0 closed

    Kubernetes issues/PRs that have lifecycle/rotten + triage/accepted:
    - total: 90 open, 273 closed
    - kubernetes/kubernetes: 63 open, 180 closed
    - kubernetes/website: 13 open, 75 closed
    - kubernetes-sigs/kubebuilder: 7 open, 5 closed
    - kubernetes/kubectl: 2 open, 4 closed
    - kubernetes-sigs/kustomize: 0 open, 5 closed
    - kubernetes-sigs/controller-runtime: 0 open, 2 closed

    Kubernetes issues/PRs that have lifecycle/stale + needs-triage:
    - total: 140 open, 86 closed
    - kubernetes/kubernetes: 103 open, 65 closed
    - kubernetes/website: 36 open, 20 closed
    - kubernetes/kubectl: 1 open, 1 closed

    Kubernetes issues/PRs that have lifecycle/rotten + needs-triage:
    - total: 78 open, 462 closed
    - kubernetes/kubernetes: 62 open, 327 closed
    - kubernetes/website: 16 open, 120 closed
    - kubernetes/kustomize: 0 open, 10 closed
    - kubernetes/kubectl: 0 open, 4 closed
    - kubernetes/kubebuilder: 0 open, 1 closed

Reply:
I'm planning to remove this change unless anyone speaks up in favor of it